cmd/watch.go

```diff
@@ -79,6 +79,13 @@ func watch(ctx *cmdutil.Ctx, events chan file.Event, sig chan os.Signal) error {
 func perform(ctx *cmdutil.Ctx, path string, op file.Op) {
 	defer ctx.DoneTask()
 
+	if op == file.Skip {
+		localAsset, _ := shopify.ReadAsset(ctx.Env, path)
+		checksumOutput := "Checksum: " + localAsset.Checksum
+		ctx.Log.Printf("[%s] Skipping %s (%s)", colors.Green(ctx.Env.Name), colors.Blue(path), checksumOutput)
+		return
+	}
+
```

Comment:
The remove if statement could just be an `else if` now and then you don't need that return.

Reply:
How about using `case`? 9c586d2

Reply:
Sure, looks good!